framework/doc/content/source/interfaces/Coupleable.md

```diff
@@ -1,31 +1,62 @@
-adCoupledGradient | Gradients of a coupled variable in q-points with automatic differentiation info
-adCoupledSecond | Second derivatives of a coupled variable in q-points with automatic differentiation info
-coupledNodalValue | Values of a coupled variable at nodes.
-adCoupledNodalValue | Values of a coupled (vector) variable at nodes with automatic differentiation info
-coupledVectorValue | Values of a coupled vector variable in q-points
-adCoupledVectorValue | Values of a coupled vector variable in q-points with automatic differentiation info
-coupledCurl | Curl of a coupled vector variable in q-points
-coupledDot | Time derivative of a coupled variable
-coupledDotDu | Derivative of a time derivative of a coupled variable
-coupledNodalDot | Nodal value of the time derivative of a coupled variable
-coupledVectorDot | Time derivative of a coupled vector variable
-coupledVectorDotDu | Derivative of a time derivative of a coupled vector variable
-adCoupledLowerValue | Value a coupled lower-dimensional variable with automatic differentiation info
-
-For values, gradients and second derivatives, users can request old and older values in case they are running a transient simulation.
-In case of old and older values, the methods are called `coupledValueOld` and `coupledValueOlder`, respectively.
-Similarly,
+This class provides an API for coupling different kinds of variables values into MOOSE systems.
+The following tables summarize the methods it provides.
+
+| Methods for scalar field variables | Description |
+| :--- | :--- |
+`coupledValue`*‡§ | Value of a coupled variable at q-points
+`coupledGradient`*‡§ | Gradient of a coupled variable at q-points
+`coupledSecond`*‡§ | Second spatial derivatives of a coupled variable at q-points
+`coupledDot`†§ | Time derivative of a coupled variable at q-points
+`coupledDotDot`†§ | Second time derivative of a coupled variable at q-points
+`coupledDotDu` | Derivative with regards to the variable of the time derivative of a coupled variable at q-points
+`coupledDotDotDu` | Derivative with regards to the variable of the second time derivative of a coupled variable at q-points
+`coupledGradientDot`§ | Time derivative of the gradient of a coupled variable at q-points
+`coupledGradientDotDot` | Second time derivative of the gradient of a coupled variable at q-points
+
+
+| Methods for nodal scalar field variables | Description |
+| :--- | :--- |
+`coupledNodalValue`*‡§ | Value of a coupled variable at nodes
+`coupledNodalDot`† | Time derivative of a coupled variable at nodes
+`coupledNodalDotDot`† | Second time derivative of a coupled variable at nodes
+
+
+| Methods for vector field variables | Description |
+| :--- | :--- |
+`coupledVectorValue`‡§ | Value of a coupled vector variable at q-points
+`coupledVectorGradient`‡§ | Gradient of a coupled vector variable at q-points
+`coupledCurl`‡ | Curl of a coupled vector variable at q-points
+`coupledDiv`‡ | Divergence of a coupled vector variable at q-points
```

Comment:
Hi @GiudGiud, can we please get a third opinion on this?
It just doesn't feel right to me.

I've never come across this definition:
> A div of a standard field variable is a thing: dudx + dudy + dudz

For me, that's just $\nabla u \cdot (1, 1, 1)$, but I've never seen it named "divergence" anywhere.

Now, if the argument is to rename to `coupledVectorDiv` because it acts on vector variables, that's fine. But in that case I'd also prefer to rename `coupledCurl` to `coupledVectorCurl`.

Reply:
oh yeah my bad

Reply:
I was thinking of EM and thought I remembered E was not a vector field. But it is